### DISCUSSION TITLE
Flameshot has a huge delay, when making screenshots on KDE Plasma

### DISCUSSION THREAD

**Discussion by Gigas002** [flameshot-org/flameshot#3032] [Q&A] [answered]
Using manjaro linux, package [flameshot-git](https://aur.archlinux.org/packages/flameshot-git). Happens on both x11 and wayland sessions. This is not an issue on Windows, since it does screenshots the moment I press the `Print` button. Thought it creates a new process each time the `Print` button is pressed, monitored processes by calling `flameshot`, switching to `tty2` and looking through `htop`, but no new processes seems to be created, so I guess it's not an issue.
Is there something I am missing and should have done in configuration?

Upd. The issue seems to happen only when `flameshot` is called using `Print` button. When calling screenshot function from terminal (`flameshot gui`) or by clicking the tray icon it happens instantly, as it should.

Upd 2. Monitored the `htop` during process, pressing the `Print` button creates a new process with `--delay 500` argument.

Upd 3. Remade the shortcuts and it seems to works as should. Probably just set them by myself and missed something the previous time. The issue is resolved.

**Comment by ElonVampire** [comment]
收到！

**Reply by mmahmoudian** [reply]
@ElonVampire  Please do not post such things. it will be considered as spam. If you have something to add to the discussion, comment, otherwise, please avoid making comments. Thank you.

**Comment by mmahmoudian** [comment, marked as answer]
@Gigas002 Thanks for providing the updates. This is crucial process and help others to debug their situation step-by-step as well.

At some point in the past (I think before I become one of the maintainers of the project), we had that `--delay 500` in KDE shortcuts, but we have updated the keyboard shortcuts some years back. more info can be found here:

https://flameshot.org/docs/guide/key-bindings/#on-kde-plasma-desktop

Cheers,
